fixtures/lint/casing-properties-fail.yaml

```diff
@@ -21,13 +21,13 @@ paths:
       tags:
         - Test
       responses:
-        '200':
+        "200":
           description: Response description
           content:
             application/json:
               schema:
                 type: object
                 properties:
-                  foo_bar:
+                  fooBar:
```

Comment:
Apologies if I'm missing the point, but why wouldn't this be snake case, as well?

Reply:
This is a failure test case so making it camel case will make the test (correctly) fail.